src/WikibaseRdfExtension.php

```diff
@@ -73,4 +74,36 @@ public function newMappingRdfBuilder( RdfWriter $writer ): MappingRdfBuilder {
 		);
 	}
 
+	public function newEntityIdParser(): EntityIdParser {
+		return new BasicEntityIdParser();
+	}
+
+	public static function getMappingsApiFactory(): GetMappingsApi {
```

Comment:
Thoughts on naming this? Since we're not creating a factory object it sounds odd to call it `new...Factory` or `create...Factory`. But on the other hand when the name starts with `get...` or `save...` it sounds like we're doing something.